crates/ruff_python_parser/src/parser/mod.rs

```diff
@@ -1100,7 +1114,8 @@ bitflags! {
         const EXCEPT = 1 << 3;
         const ASSIGNMENT_TARGETS = 1 << 4;
         const TYPE_PARAMS = 1 << 5;
-        const IMPORT_FROM_AS_NAMES = 1 << 6;
+        const IMPORT_FROM_AS_NAMES_PARENTHESIZED = 1 << 6;
+        const IMPORT_FROM_AS_NAMES_UNPARENTHESIZED = 1 << 29;
         const IMPORT_NAMES = 1 << 7;
```

Comment:
I recommend updating all the numbers. It's a bit painful but it otherwise becomes difficult to spot what the next free bit is.
```suggestion
        const IMPORT_FROM_AS_NAMES_PARENTHESIZED = 1 << 6;
        const IMPORT_FROM_AS_NAMES_UNPARENTHESIZED = 1 << 7;
        const IMPORT_NAMES = 1 << 8;
```

Reply:
Another alternative (considering that it is a bitflag) is to have a flag `PARENTHESIZED` and for `IMPORT_FROM_AS_NAMES_PARENTHESIZED` you set `IMPORT_FROM_AS_NAMES` and `PARENTHESIZED`

Reply:
Yeah, I like the alternative, will make it as a follow-up for other variants as well.